src/test/java/sign_up/interface_adapters/SignUpViewModelTest.java

```diff
@@ -0,0 +1,149 @@
+package sign_up.interface_adapters;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
```

Comment:
I think the issues with your test files come from importing from `org.junit.jupiter.api` instead of just `org.junit` (without the `jupiter.api`. For example use `import org.junit.Test` instead.

Reply:
omg that was actually it. Thank you sm